pilot/pkg/xds/discovery.go

```diff
@@ -391,9 +400,17 @@ func debounce(ch chan *model.PushRequest, stopCh <-chan struct{}, opts debounceO
 		if eventDelay >= opts.debounceMax || quietTime >= opts.debounceAfter {
 			if req != nil {
 				pushCounter++
-				log.Infof("Push debounce stable[%d] %d: %v since last change, %v since last push, full=%v",
-					pushCounter, debouncedEvents,
-					quietTime, eventDelay, req.Full)
+				configsUpdated := len(req.ConfigsUpdated)
+				if configsUpdated == 1 {
```

Comment:
@howardjohn This kind of code looks really urgly and unreadable. And could not tell how useful it is.

Reply:
Can you give constructive feedback on his to improve it instead of calling it ugly? Ugly is subjective and to me it's fine, so I am not sure how to improve. Thanks!